### PR TITLE
poco: clang update fix

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-poco/007-mingw-wide-character-startup-module.patch [hunk 1]
```diff
@@ -4,7 +4,7 @@
  	DEFINE_SYMBOL Util_EXPORTS
  )
  
-+if(MINGW)
++if(MINGW OR CLANG)
```
Comment (lazka) [inline mingw-w64-poco/007-mingw-wide-character-startup-module.patch:7]: That shouldn't change anything

File: mingw-w64-poco/011-cmake-mingw.patch [hunk 2]
```diff
@@ -15,7 +15,7 @@
  			endforeach()
  		endif(X64)
  	endif()
-+	if(MINGW)
++	if(MINGW OR CLANG)
```
Comment (lazka) [inline mingw-w64-poco/011-cmake-mingw.patch:18]: same

Comment (raedrizqie) [reply mingw-w64-poco/011-cmake-mingw.patch:18]: i dont know, but without this, cmake couldnt find windmc.exe

**Comment by lazka** [comment]
It seems to build fine in #10194. Can I merge that instead?

**Comment by raedrizqie** [comment]
> It seems to build fine in #10194. Can I merge that instead?

of course..

**Pull request closed without merging**